src/VisualStudio/IntegrationTest/TestUtilities/InProcess/StartPage_InProc.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.InProcess
+{
+    internal class StartPage_InProc : InProcComponent
+    {
+        public static StartPage_InProc Create()
+            => new StartPage_InProc();
+
+        public bool IsEnabled()
+        {
+            return InvokeOnUIThread(() =>
+            {
+                var property = GetProperty();
+                return (int)property.Value == 4;
+            });
+        }
+
+        public void SetEnabled(bool enabled)
+        {
+            InvokeOnUIThread(() =>
+            {
+                var property = GetProperty();
+                property.Value = enabled ? 5 : 4;
```

Comment:
Where do these numbers come from?

Reply:
![image](https://user-images.githubusercontent.com/1408396/49986511-dbc0df00-ff35-11e8-9b2f-6e140d6610fd.png)

Reply:
@sharwell Can we please add a comment?

Reply:
➡️ Now updated